digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/TierekisteriDataImporter.scala

```diff
@@ -304,27 +332,29 @@ object TierekisteriDataImporter {
       }
     }
 
-    if(args.size < 2){
+    if(args.length < 2){
       println("Usage: TierekisteriDataImporter <operation> <assetType> [<args>]")
     }else{
       val operation = args(0)
       val assetType = args(1)
 
-      val availableAssetTypes = tierekisteriDataImporters.keySet ++ Set("trafficVolume")
+      val availableAssetTypes = tierekisteriDataImporters.keySet ++ tierekisteriDataConverter.keySet ++ Set("trafficVolume")
 
       if(availableAssetTypes.contains(assetType)){
         operation match {
           case "import" =>
             if(assetType == "trafficVolume")
               importTrafficVolumeAsset(tierekisteriTrafficVolumeAssetClient)
             else
-              importAssets(tierekisteriDataImporters.get(assetType).get)
+              importAssets(tierekisteriDataImporters(assetType))
           case "update" =>
             val lastExecutionDate = getDateFromArgs(args)
             if(assetType == "trafficVolume")
               println("The asset type trafficVolume doesn't support update operation.")
             else
-              updateAssets(tierekisteriDataImporters.get(assetType).get, lastExecutionDate)
+              updateAssets(tierekisteriDataImporters(assetType), lastExecutionDate)
+          case "converter" =>
+              importAssets(tierekisteriDataConverter(assetType))
```

Comment:
Wouldn't it be better to add a default case?

Reply:
Will probably throw an exception if no match. but we could print something like: 
`case _ => println("Usage: import | update | converter")`

Reply:
on the other hand, on the dropdown we only have those options so...